runtime/commonMain/src/kotlinx/serialization/Serializers.kt

```diff
@@ -92,7 +105,7 @@ public fun serializer(type: KType): KSerializer<Any?> {
  *
  * @throws SerializationException if serializer can't be found.
  */
-@UnsafeSerializationApi
+@InternalSerializationApi
```

Comment:
Not sure why this methods should be internal instead of unsafe. There is still possibility that one does not have `KType`/`j.l.r.Type`, but have `KClass`